plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -1175,6 +1177,160 @@ public function get_global( string $global_name ) {
 		return wc_get_container()->get( LegacyProxy::class )->get_global( $global_name );
 	}
 
```

Comment:
🤔 I'm wondering if it might make sense to keep all this code for setting up the stub in a separate file (maybe in the `Internal` namespace, even?). Seems like it adds a lot of clutter to the main, public WooCommerce class for something that will increasingly be an irrelevant edge case.

Reply:
That's a very good idea! I have moved the stub to a dedicated internal class, `LegacyRestApiStub` in `src/Internal/Utilities`.